AUTHORS

```diff
@@ -0,0 +1,3 @@
+See root_files/humans.txt
+
+Please add yourself when you contribute code if you like.
```

Comment:
this isn't a markdown file. Could make it one I guess. Figured it would just be txt.